nanoleaf_client.py

```
@@ -645,6 +655,123 @@ def identify(self, device: NanoleafDevice) -> None:
             raise NanoleafError('Device is not paired')
         _http_json('PUT', f'{device.base_url}/{device.auth_token}/identify')
 
+    def identify_panel(self, device: NanoleafDevice, panel_id: int, wait: bool = False) -> None:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)